docs/installation.md

```diff
@@ -5,9 +5,9 @@ and quay.io for the public registry. We also test on crc version v1.7.0+.
 
 ### Supported versions
 * [OpenShift® Container Platform](https://www.openshift.com/products/container-platform/)
-  * Fully supported: 4.3
+  * Fully supported: 4.3 and later
```

Comment:
"Tested on" instead of "Fully supported"? I'm just always hesitant to use the word supported.